content/pop-basics.md

```diff
@@ -17,30 +17,169 @@ section: pop
+
+This video shows the features of Pop!\_Shell auto-tiling. Covered are default behaviors, and ways to adjust settings and view keyboard shortcuts.
+
+- Click the Tiling icon in the top right of top-bar. The icon is to the left the the `Top-right Menu`.
+- The dropdown features options to automatically tile/untile the current application windows.
+- `Active Hints` can be enabled which highlight the active application window in a color of your choice.
+- Users can adjust the gap between application windows, and view keyboard shortcuts.
+
+**NOTE:** Current versions of Pop!\_Shell also offer options for showing or hiding window title bars.
+
+### Keyboard Shortcuts
+
+<iframe width="900" height="500" src="https://www.youtube.com/embed/aqj0cRTZaVE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+This video covers keyboard shortcuts for opening the `Launcher`, opening applications, and switching between open applications or workspaces. It also covers how to navigate and adjust open applications windows.
+
+For a more detailed overview of Pop!\_OS keyboard shortcuts, refer to [this article](/articles/pop-keyboard-shortcuts)
+
+## Workspaces
+
+Depending on which version of Pop!\_OS is installed on your system, the placement and appearance of workspaces may vary. Usage and features of workspaces remain essentially the same across versions.
+
+### Pop!_OS 21.04 COSMIC Desktop - Workspaces Tutorial
+
+<iframe width="900" height="500" src="https://www.youtube.com/embed/USQjxLP1Z5k" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+This video shows how to access the `Workspaces` overview and how to use gestures and keyboard shortcuts to navigate COSMIC.
+
+There are four ways to access the `Workspaces` overview:
 
-To navigate within the desktop, either click the <u>Activities</u> button in the top left, or press the <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon></kbd> key on the keyboard.  This will show an overview of all open windows and provide a text box to search your system.
+1. Click on the Workspaces icon in the Dock.
+2. Click on `Workspaces` in the top-left.
+3. Swipe left on the trackpad with four (4) fingers.
+4. Press <kbd>SUPER</kbd> + <kbd>D</kbd>
+
+Once in the `Workspaces` overview, you can move active application windows to different workspaces by clicking and dragging them, or by pressing - <kbd>SUPER</kbd> + <kbd>SHIFT</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>.
+
+Click on the thumbnail for a specific workspace to switch to it, or use
+<kbd>SUPER</kbd> + <kbd>CTRL</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd> to switch to workspaces above or below the current one.
```

Comment:
I think this should be Super+Enter then press these arrow keys to move windows.

Reply:
Fixed.